include/lib.h

```diff
@@ -303,6 +303,7 @@ static inline void ud2(void) { asm volatile("ud2"); }
 #define BUG()                                                                            \
     do {                                                                                 \
         ud2();                                                                           \
+        while(true) {}                                                                   \
     } while (0)
```

Comment:
changing `while(0)` to `while(true)` would be easier

Reply:
if that's the wanted change, I can do that as well, yes. Let me iterate.